### DISCUSSION TITLE
Do we change to `npx @react-native-community/cli` in future?

### DISCUSSION THREAD

**Discussion by pnthach95** [react-native-community/cli#2403] [General]
https://reactnative.dev/docs/getting-started-without-a-framework

New document doesn't use `npx react-native`. So, will it be discontinued in future and change to `@react-native-community/cli`?

**Comment by szymonrybczak** [comment]
Yes, after initiative of React Native Frameworks landed you should be using `npx @react-native-community/cli init` to init project with bare React Native. We'll have a better package name for that soon :)
